test_runner/regress/test_sharding.py

```diff
@@ -177,7 +177,16 @@ def test_sharding_split_unsharded(
     env.storage_controller.consistency_check()
 
 
-def test_sharding_split_compaction(neon_env_builder: NeonEnvBuilder):
+@pytest.mark.parametrize(
+    "failpoint",
+    [
+        None,
+        "compact-shard-ancestors-localonly-kill",
+        "compact-shard-ancestors-enqueued-kill",
+        "compact-shard-ancestors-persistent-kill",
```

Comment:
We don't need to name the failpoint for the action. Plain `return` would fail for these failpoints when defined with `fail!(name)`.